book/website/ethical-research/activism/activism-case-study-google.md

```diff
@@ -0,0 +1,53 @@
+[Source, accessed 8 December 2020](https://jacobinmag.com/2018/06/google-project-maven-military-tech-workers)
+
+### Walkout for Real Change - 2018 
+In response to allegations of million dollar exit packages for executives accused of sexual harassment against workers, workers at Google offices across the globe walked out demanding: 
+* Ending of private arbitration in sexual harassment cases 
+* Publication of a transparency report on instances of sexual harassment
+* Further disclosures of salaries and compensation
+* An employee representative on the company board
+* A chief diversity officer who could speak directly to the board
+
+Google agreed to end the use of private arbitration in sexual harassment cases. 
+
+[Source, NYT, accessed 18 May 2021](https://www.nytimes.com/2018/11/01/technology/google-walkout-sexual-harassment.html)
+
+### No Police Contracts - 2020
+In the wake of protests at the killing of George Floyd in May 2020, over 1,100 Alphabet workers signed a petition demanding that the company stop selling technology to police departments and end existing contracts with police departments with a history of racial discrimination. 
+The company insisted that the contracts were in line with its AI Principles. 
+
+[Source, CNBC, accessed 18 May 2021](https://www.cnbc.com/2020/06/22/google-employees-petition-company-to-cancel-police-contracts.html)
+
+(er-activism-case-study-google-union)=
+## Alphabet Worker's Union
+
+The work that went into speaking out against different issues at Google has now transformed into a new union, specifically for people employed at Google or one of the other companies owned by Alphabet.
+The union was launched in January 2021 and has over 800 members as of May 2021. 
+Importantly, all workers at Alphabet companies, whether they are cleaners or tech workers, on fixed-term contracts or in permanent employment, are eligible to join the union.
+This broad membership makes it much easier for the workers to organise together and fight for better conditions for everyone in the workplace, rather than segregating people based on the type of work they do. 
+It also reinforces that everyone in the workplace is important and has a part to play in the union, regardless of what their day-to-day role looks like. 
+
+The Alphabet Union is supported by a larger union, [Communications Workers of America](https://cwa-union.org/), who are also [campaigning to help workers in other tech and digital companies to organise in unions](https://cwa-union.org/news/releases/cwa-launches-new-initiative-in-support-of-organizing-tech-and-game-workers). 
+
+## Further Resources 
+
+* [Alphabet Worker's Union](https://alphabetworkersunion.org/)
+* [Communication Worker's Union - a UK-based union with a dedicated tech branch](https://cwu.org) 
+* 
```

Comment:
Can we link to TTW's Unionisation page :) ?

Reply:
Yes, please add the link in! I can't find it 😫